### PR TITLE
As discussed, generate an easy-to use for demo config

### PR DESCRIPTION
**Pull request by costinm**
istio-demo.yaml and istio-demo-auth.yaml replace the old istio.yaml.
Default to:
- v2/alpha3
- include all addons needed by bookinfo

The old unmaintained addons are removed from release (zipkin is still around because of a test).

People interested in the 'easy to install' mode should do more manual testing.

### CODE REVIEW

**Comment by costinm** [comment]
On removed stuff: it was redundant with the defaults in values.yaml, easier to read if the files contain only
overrides.

File: release/create_release_archives.sh [hunk 1]
```diff
@@ -131,10 +131,15 @@ for unwanted_manifest in \
     istio-one-namespace-auth.yaml \
     istio-multicluster.yaml \
     istio-auth-multicluster.yaml \
+    istio.yaml \
+    addons/zipkin.yaml \
```
Comment (rshriram) [inline release/create_release_archives.sh]: See @douglas-reid 's PR for Jaeger.. I guess instead of Zipkin, we will have Jaeger..

**Comment by rshriram** [comment]
What exactly was discussed/what was the resolution?

**Comment by sdake** [comment]
@rshriram In the environments wg meeting on Wednesday, there was a long and hard to follow debate about what we ship vs don't ship related to istio.yaml.

My understanding of the decision was that:
* We should ship a demo istio.yaml for folks wanting to demonstrate latest and greatest features (v3 + addons) + bookinfo or other types of demo scenarios.  This demo yaml would not be supported - just available.
* We would use ictiolctl gendeploy (or helm template) for folks wanting to manage their own deployment content.  istioctl gendeploy is required because some folks didn't want to take a hard dep on helm for the `template` operation.
* We would (Costin did in this PR) sanitize as much of the default settings as possible prior to 0.8
* Post 0.8 we would do a more complete evaluation of  the options in values.yaml and move/delete as needed

If I missed anything or misrepresented anything (other folks in the env wg meeting) - please feel free to correct.

Cheers
-steve

File: install/updateVersion.sh [hunk 2]
```diff
@@ -56,7 +56,6 @@ function gen_file() {
 
 if [[ ! -z ${ISTIO_RELEASE:-} ]]; then
     for target in istio-demo.yaml istio-demo-auth.yaml; do
-        echo XXXXXXXXXXXXXXX $target
```
Comment (sdake) [inline install/updateVersion.sh]: I caught this in the original review, although thought you wanted it in for some reason :)

File: install/updateVersion_orig.sh [hunk 3]
```diff
@@ -274,11 +274,8 @@ function update_istio_addons() {
   mkdir -p $DEST
   pushd $TEMP_DIR/templates/addons
   execute_sed "s|image: {MIXER_HUB}/\(.*\):{MIXER_TAG}|image: ${MIXER_HUB}/\1:${MIXER_TAG}|" grafana.yaml.tmpl
-  execute_sed "s|image: {MIXER_HUB}/\(.*\):{MIXER_TAG}|image: ${MIXER_HUB}/\1:${MIXER_TAG}|" servicegraph.yaml.tmpl
   sed "s|{ISTIO_NAMESPACE}|${ISTIO_NAMESPACE}|" grafana.yaml.tmpl  > $DEST/grafana.yaml
```
Comment (sdake) [inline install/updateVersion_orig.sh]: is this what was needed for one of the ci/cd test cases?

Would be nice if we could remove updateVersion_orig.sh in master once 0.8 unfreezes.

File: install/updateVersion.sh (hunk 2 above)
Comment (costinm) [reply install/updateVersion.sh]: Debug statement. The generation of the release needs cleanup...

File: install/updateVersion_orig.sh (hunk 3 above)
Comment (costinm) [reply install/updateVersion_orig.sh]: Agreed, but there are still tests that depend on stuff.

**Comment by ldemailly** [comment]
how do I get v1 istio-auth.yaml now ?

(`make istio-auth.yaml` generates a file with v2 proxy - we need to keep a "0.7.1" compatible way to get those files - and in short term I need it for the daily perf run before also adding a new v2 perf run)
